src/GovUk.Education.ExploreEducationStatistics.Admin/ViewModels/DataReplacementPlanViewModel.cs

```diff
@@ -1,49 +1,43 @@
-﻿#nullable enable
-using System;
-using System.Collections.Generic;
-using System.Linq;
+#nullable enable
 using GovUk.Education.ExploreEducationStatistics.Common.Converters;
 using GovUk.Education.ExploreEducationStatistics.Common.Extensions;
 using GovUk.Education.ExploreEducationStatistics.Common.Model;
 using GovUk.Education.ExploreEducationStatistics.Common.Utils;
 using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace GovUk.Education.ExploreEducationStatistics.Admin.ViewModels
 {
     public class DataReplacementPlanViewModel
     {
-        public IEnumerable<DataBlockReplacementPlanViewModel> DataBlocks { get; }
-        public IEnumerable<FootnoteReplacementPlanViewModel> Footnotes { get; }
-        public Guid OriginalSubjectId { get; }
-        public Guid ReplacementSubjectId { get; }
-
-        public DataReplacementPlanViewModel(
-            IEnumerable<DataBlockReplacementPlanViewModel> dataBlocks,
-            IEnumerable<FootnoteReplacementPlanViewModel> footnotes,
-            Guid originalSubjectId,
-            Guid replacementSubjectId)
-        {
-            DataBlocks = dataBlocks;
-            Footnotes = footnotes;
-            OriginalSubjectId = originalSubjectId;
-            ReplacementSubjectId = replacementSubjectId;
-        }
+        public IEnumerable<DataBlockReplacementPlanViewModel> DataBlocks { get; init; } = [];
+        public IEnumerable<FootnoteReplacementPlanViewModel> Footnotes { get; init; } = [];
+        public bool IsLinkedToApiDataSetVersion { get; init; }
```

Comment:
Can this just be a derived field by checking if DeleteApiDataSetVersionPlanViewModel is null or not?

Reply:
I've now removed this field in [EES-5192](https://github.com/dfe-analytical-services/explore-education-statistics/pull/4958) :)